frontend/rolecall/src/app/services/super-ballet-display.service.spec.ts

```diff
@@ -0,0 +1,13 @@
+import {SuperBalletDisplayService} from './super-ballet-display.service';
+
+describe('SuperBalletDisplayService', () => {
+  let service: SuperBalletDisplayService;
+
+  beforeEach(() => {
+    service = new SuperBalletDisplayService;
```

Comment:
If you run `ng test` you will get a failure with this file due to missing parenthesis. Always make sure your tests pass :)

Reply:
Thanks. Fixed.